### PR TITLE
Translation fixes for Brazilian Portuguese for Groovy views

### PR DESCRIPTION
**Pull request by glasswalk3r**
No Jira issue related.

### Testing done

- Checked with Jenkins::i18n for missing/repeated entries
- Checked with Jenkins::i18n for entries with `TranslateMe` that were ignored
- Compiled and manually navigated over the Jenkins views

### Proposed changelog entries

- Added missing translations for Jenkins views based on Groovy for pt_BR.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7513"></a>

### CODE REVIEW

File: cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties
Comment (daniel-beck) [inline cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties:23]: Not a localization resource bundle, remove.

File: core/src/filter/resources/hudson/model/hudson-version_pt_BR.properties
Comment (daniel-beck) [inline core/src/filter/resources/hudson/model/hudson-version_pt_BR.properties:23]: Not a localization resource bundle, remove.

File: core/src/filter/resources/jenkins/model/jenkins-version_pt_BR.properties
Comment (daniel-beck) [inline core/src/filter/resources/jenkins/model/jenkins-version_pt_BR.properties:24]: Not a localization resource bundle, remove.

File: core/src/filter/resources/jenkins/slaves/remoting-info_pt_BR.properties
Comment (daniel-beck) [inline core/src/filter/resources/jenkins/slaves/remoting-info_pt_BR.properties:24]: Not a localization resource bundle, remove.

File: core/src/main/resources/commons-logging_pt_BR.properties
Comment (daniel-beck) [inline core/src/main/resources/commons-logging_pt_BR.properties:23]: Not a localization resource bundle, remove.

File: cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties
Comment (glasswalk3r) [reply cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties:23]: Will do.
But how can I automatically identify that a Properties is a resource bundle and not localization?
By this example, I could try to find the value beginning with `${` and ending with `}`?

File: core/src/main/resources/commons-logging_pt_BR.properties
Comment (glasswalk3r) [reply core/src/main/resources/commons-logging_pt_BR.properties:23]: Will do, but how can I automatically define that by value this is a resource bundle and not a internationalization Properties?
By the parent directory?

File: cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties
Comment (glasswalk3r) [reply cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties:23]: Or it would be by the parent directory?

Comment (glasswalk3r) [reply cli/src/filter/resources/jenkins/cli/jenkins-cli-version_pt_BR.properties:23]: This [doc reference](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/ResourceBundle.html) didn't help much.

File: core/src/filter/resources/hudson/model/hudson-version_pt_BR.properties
Comment (glasswalk3r) [reply core/src/filter/resources/hudson/model/hudson-version_pt_BR.properties]: Done.

File: core/src/filter/resources/jenkins/model/jenkins-version_pt_BR.properties
Comment (glasswalk3r) [reply core/src/filter/resources/jenkins/model/jenkins-version_pt_BR.properties]: Done.

File: core/src/filter/resources/jenkins/slaves/remoting-info_pt_BR.properties
Comment (glasswalk3r) [reply core/src/filter/resources/jenkins/slaves/remoting-info_pt_BR.properties]: Done.

**Review by NotMyFault — APPROVED** [review]

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
